src/index.js

```diff
@@ -10,10 +13,22 @@ export default function livereload (options = { watch: '' }) {
     options.watch = options.watch || ''
   }
 
+  // release previous server instance if rollup is reloading configuration in watch mode
+  if (server) {
+    server.close()
+    server = null
```

Comment:
This line is unneeded as server is reassigned on line 31.

Reply:
I figured it could crash in between. We might as well give it a chance to recover on next run... I can remove it if you prefer, no problem.